lib/plugins/aws/package/lib/strip-null-props-from-template-resources.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     const resources = this.serverless.service.provider.compiledCloudFormationTemplate.Resources;
 
     for (const resource of Object.values(resources)) {
-      if (resource.Properties) {
+      if (resource.Properties && resource.Properties !== null) {
```

Comment:
This addition has no effect (`null` is a falsy value)

Reply:
I've looked closer and I  believe expected to fix, is to ignore any `null` properties on `resource.Properties` object (and not ignore `resource.Properties` being null)